web/api/v1/api.go

```diff
@@ -1219,6 +1220,7 @@ func (api *API) rules(r *http.Request) apiFuncResult {
 			Rules:          []Rule{},
 			EvaluationTime: grp.GetEvaluationTime().Seconds(),
 			LastEvaluation: grp.GetLastEvaluation(),
+			Limit:          grp.Limit(),
```

Comment:
The `Limit` field should go after the `Interval` field, as is in the configuration documentation.